src/python/bot/fuzzers/afl/launcher.py

```diff
@@ -847,14 +776,55 @@ def check_error_and_log(error_regex, log_message_format):
     while self.should_try_fuzzing(max_total_time, num_retries):
       # Increment this now so that we can just "continue" without incrementing.
       num_retries += 1
-      # If we have spent a while trying to fuzz and are still retrying, then
-      # make sure we use AFL_FAST_CAL, otherwise we won't have much time for
-      # fuzzing.
-      self.strategies.decide_fast_cal_manual(get_time_spent_fuzzing())
-      if self.strategies.use_fast_cal:
-        environment.set_value(constants.FAST_CAL_ENV_VAR, 1)
 
       self.afl_setup()
+
+      # If the target was compiled for CMPLOG we need to set this.
+      build_directory = environment.get_value('BUILD_DIR')
+      if not build_directory:  # Required for passing the unit tests.
+        build_directory = '/out'
```

Comment:
We don't do things like this where we set a default in prod code just for the purpose of unittests. I'm going to try to fix this myself by editing this PR.

Reply:
ok, good luck :)